src/components/yeti/map-layers/simplify.js

```diff
@@ -0,0 +1,124 @@
+/*
```

Comment:
why is i copied, and not importing the npm pakage instead?

Reply:
Well, it seems that package is 2d only, and it uses object values (x/y) instead of list. The script is not configurable to maintain performance. The advice is to copy/paste and update as needed.

See http://mourner.github.io/simplify-js/
> Configurability of point format would draw a significant performance overhead, so if you use a different format, e.g. [Number, Number], just run search/replace of .x and .y through the source of the library to suite it to your application. For a version that works with 3D points, check out the 3d branch.

Reply:
OK, didn't notice it. Fine then!